web/webapp_content/app/authentication/authentication.service.js

```diff
@@ -5,20 +5,25 @@
         .module('org.perfrepo.authentication')
         .service('authenticationService', AuthService);
 
-    function AuthService($http, API_URL, $state, $window, $q) {
+    function AuthService($http, $location, API_URL, $window, $q) {
         var vm = this;
         vm.login = login;
         vm.logout = logout;
         vm.isAuthenticated = isAuthenticated;
         vm.getToken = getToken;
         vm.getUser = getUser;
+        vm.redirectToLogin = redirectToLogin;
+        vm.saveAttemptLocation = saveAttemptLocation;
+        vm.redirectToAttemptedLocation = redirectToAttemptedLocation;
+        vm.location = '/dashboard';
```

Comment:
Could you move all these paths (/dashboard, /login etc.) to some centralized location like you do it in app.module with .constant('API_GROUP_URL', 'rest/json/groups')? I would be great to not have it spread all around the frontend.